app/views/userProfile.scala.html

```diff
@@ -79,6 +79,26 @@
                                 <td id="map-legend-audited-street"><svg><path d="M 2 10 L 18 10 z"></path></svg></td>
                                 <td>@Messages("audited.street")</td>
                             </tr>
+                            <tr>
+                                <td id="map-legend-correct"><img src='@routes.Assets.at("javascripts/SVValidate/img/Checkmark.png")' style="width: 23px; padding: 3px;"></td>
```

Comment:
Check that all of these stylings are necessary. I think that you have code that was copied from the old version of LabelMap legend code, but I made some updates so that it's more simple and some of those changes haven't been included. For example, I don't think you need to include `style="width: 23px; padding: 3px;"` here, I believe that I've moved that to a CSS file.

For anything that was copied from LabelMap, just double check that it hasn't been updated on the LabelMap side since you started working on this!